### PR TITLE
MODFQMMGR-368-fixes: changing the dash separator

### PR DESCRIPTION
**Pull request by kjain110**

### CODE REVIEW

File: src/main/java/org/folio/fqm/service/LocalizationService.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ private String getSourceTranslationPrefix(EntityType entityType, String columnNa
     if (currentSourceIndex > 0) {
       String currentSource = columnName.substring(0, currentSourceIndex);
       String formattedKey = ENTITY_TYPE_COLUMN_AND_SOURCE_LABEL_TRANSLATION_TEMPLATE.formatted(entityType.getName(), currentSource);
-      return  translationService.format(formattedKey) + " - ";
+      return  translationService.format(formattedKey) + " — ";
```
Comment (kjain110) [inline src/main/java/org/folio/fqm/service/LocalizationService.java:72]: is the dash used here is correct ?

Comment (mweaver-ebsco) [reply src/main/java/org/folio/fqm/service/LocalizationService.java:72]: If I'm reading MODFQMMGR-368 correctly, the separator needs to be localized and the `en` version should be the em dash

Comment (ncovercash) [reply src/main/java/org/folio/fqm/service/LocalizationService.java:72]: monospace is hard:
      return  translationService.format(formattedKey) + " - ";
      return  translationService.format(formattedKey) + " — ";

it's good

**Review by ncovercash — APPROVED** [review]

File: src/main/java/org/folio/fqm/service/LocalizationService.java (hunk 1 above)
Comment (ncovercash) [reply src/main/java/org/folio/fqm/service/LocalizationService.java:72]: this whole thing (dash/combining) should be changed at some point though, to accommodate other languages and RTL, but that's later

**Pull request merged**